### ISSUE TITLE
[BUG] SiteHandling with different errorSource leads to error

### ISSUE THREAD

**Issue by calien666** [sudhaus7/wizard#2]
Configured Error handler in SiteConfig:

```yaml
errorHandling:
  -
    errorCode: 404
    errorHandler: PHP
    errorPhpClassFQCN: Vendor\ErrorHandling\Error\GlobalPageContentErrorHandler
```
leads to:
```sh                                                     
  [ ErrorReporting\Warning ]                         
  Warning: Undefined array key "errorContentSource"  
                                                     

Exception code: 1

Exception trace:
#0 ()
   vendor/sudhaus7/sudhaus7-wizard/Classes/CreateProcess.php:1151
#1 SUDHAUS7\Sudhaus7Wizard\CreateProcess->finalYaml()
   vendor/sudhaus7/sudhaus7-wizard/Classes/CreateProcess.php:153
#2 SUDHAUS7\Sudhaus7Wizard\CreateProcess->run()
   vendor/sudhaus7/sudhaus7-wizard/Classes/Cli/RunCommand.php:89
#3 SUDHAUS7\Sudhaus7Wizard\Cli\RunCommand->create()
   vendor/sudhaus7/sudhaus7-wizard/Classes/Cli/RunCommand.php:192
#4 SUDHAUS7\Sudhaus7Wizard\Cli\RunCommand->execute()
   vendor/symfony/console/Command/Command.php:298
#5 Symfony\Component\Console\Command\Command->run()
   vendor/symfony/console/Application.php:1040
#6 Symfony\Component\Console\Application->doRunCommand()
   vendor/helhum/typo3-console/Classes/Console/Mvc/Cli/Symfony/Application.php:189
#7 Helhum\Typo3Console\Mvc\Cli\Symfony\Application->doRunCommand()
   vendor/symfony/console/Application.php:301
#8 Symfony\Component\Console\Application->doRun()
   vendor/symfony/console/Application.php:171
#9 Symfony\Component\Console\Application->run()
   vendor/helhum/typo3-console/Classes/Console/Core/Kernel.php:118
#10 Helhum\Typo3Console\Core\Kernel->handle()
   vendor/helhum/typo3-console/Scripts/typo3-console.php:18
#11 {closure}()
   vendor/helhum/typo3-console/Scripts/typo3-console.php:20
#12 require()
   vendor/helhum/typo3-console/typo3cms:3
#13 include()
   vendor/bin/typo3cms:120

sudhaus7:wizard [-i|--id ID] [-f|--force] [-m|--mapto MAPTO] [--] <mode>

Failed to run typo3cms sudhaus7:wizard next -v: exit status 1
```

**Comment by foppelfb** [comment]
Fixed in 30ee212c4387910e2895dfb293e346979d2f497f 
please check

**Comment by calien666** [comment]
Works, thank you
